astropy/io/fits/hdu/hdulist.py

```diff
@@ -965,10 +965,10 @@ def info(self, output=None):
             name = self._file.name
 
         results = ['Filename: {}'.format(name),
-                   'No.    Name         Type      Cards   Dimensions   Format']
+                   'No.    Name      Ver        Type      Cards   Dimensions   Format']
 
-        format = '{:3d}  {:10}  {:11}  {:5d}   {}   {}   {}'
-        default = ('', '', 0, (), '', '')
+        format = '{:3d}  {:10}  {:10} {:11}  {:5d}   {}   {}   {}'
```

Comment:
Do we really want a width of 10 for the `Ver`? That sounds a bit excessive. I would probably say 5 is more than enough.